src/plugins/build.ts

```diff
@@ -51,9 +52,9 @@ export const ethWindow: Window = global.window;
 
 export const ZK_API_BASE: string = process.env.ZK_SPECIAL_API ? process.env.ZK_SPECIAL_API : `${ETHER_PREFIX_MINUS}api.zksync.io`;
 export const ZK_NETWORK: string = process.env.ZK_NETWORK ? process.env.ZK_NETWORK : ETHER_NETWORK_NAME;
-export const APP_ZK_SCAN: string = process.env.ZK_SPECIAL_SCAN ? process.env.ZK_SPECIAL_SCAN : `https://${ETHER_PREFIX_DOT}zkscan.io`;
+export const APP_ZK_SCAN: string = process.env.ZK_SPECIAL_SCAN ? process.env.ZK_SPECIAL_SCAN : `https://explorer.${EXPLORER_PREFIX_DOT}aggregation.rifcomputing.net`;
```

Comment:
Sorry, are we sure we need this change? what if we set the `ZK_SPECIAL_SCAN` in the `.env` folder.
```
ZK_SPECIAL_SCAN="https://explorer.dev.aggregation.rifcomputing.net"
```

Reply:
@antomor yes, if we setup on the env file, that will not be needed. But just in case, this change will ensure to not point to zk sync explorer.